Cannot install in node 12, needs bufferutil 4.x

Installing on node12 results in build errors from `bufferutil -> nan`. Updating to `bufferutil@4.x` should fix it.

```
In file included from ../../nan/nan_new.h:189,
                 from ../../nan/nan.h:203,
                 from ../src/bufferutil.cc:7:
../../nan/nan_implementation_12_inl.h: In static member function ‘static Nan::imp::FactoryBase<v8::Function>::return_t Nan::imp::Factory<v8::Function>::New(Nan::FunctionCallback, v8::Local<v8::Value>)’:
../../nan/nan_implementation_12_inl.h:105:32: error: no matching function for call to ‘v8::Function::New(v8::Isolate*&, void (&)(const v8::FunctionCallbackInfo<v8::Value>&), v8::Local<v8::Object>&)’
                           , obj));
```

--- a/package.json
+++ b/package.json
@@ -27,8 +27,8 @@
     "yamljs": "^0.3.0"
   },
   "optionalDependencies": {
-    "bufferutil": "^3.0.5",
-    "utf-8-validate": "^4.0.2"
+    "bufferutil": "^4.0.1",
+    "utf-8-validate": "^5.0.2"
   },
   "devDependencies": {
     "lodash": "^4.17.11",

